README.md

```diff
@@ -281,7 +283,7 @@ buttons each time it creates a "_chain reaction_" which almost
 instantly _exceeds_ the "***call stack***"
 (_i.e. exhausts the allocated memory_) of the browser!
 
-The putting the `callback` in a _closure_ means we can pass a _reference_
+Putting the `callback` in a _closure_ means we can pass a _reference_
 to the `signal` (_parent/outer_) function to the `view` function.
```

Comment:
Should this be `from` the signal _to_ the view?

Reply:
I thought this would have to be either:
***from*** the `signal` (_parent/outer_) function to the `view` function

or

to the `signal` (_parent/outer_) function ***and then*** to the `view` function

in order to make most sense... I just wasn't entirely sure which way around it was (I thought it was the first).

Reply:
feel free to re-write it from scratch with a "shoshin" perspective.